go.mod

```diff
@@ -3,31 +3,23 @@ module github.com/fetchai/fetchd
 go 1.16
 
 require (
-	github.com/CosmWasm/wasmd v0.20.0
+	github.com/CosmWasm/wasmd v0.21.1-0.20220104133518-52477ead36e0
```

Comment:
That is a shame that there is not a tagged version of wasmd with v1.0.0-beta5. Do we think that a new release is imminent?

Reply:
Yes, I've already asked on Discord if we could have a release or pin on that commit. Ethan recommended using that commit for now, as the next release will be integrating cosmos SDK v0.45.